### PR TITLE
Update version to 1.11.2. Update csi-volume-group-operator sidecar to version v0.9.1

### PR DESCRIPTION
**Pull request by uriziv1**
Update version to 1.11.2. Update csi-volume-group-operator sidecar to version v0.9.1

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/bundle-1.11.2.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator-community
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.10-v4.12"
```
Comment (kasserater) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/bundle-1.11.2.Dockerfile:12]: we will support from 4.13 through 4.15

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+  namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.s390x: supported
+    operatorframework.io/arch.ppc64le: supported
+  annotations:
+    capabilities: "Seamless Upgrades"
+    categories: "Storage,Cloud Provider"
+    certified: "true"
+    containerImage: quay.io/ibmcsiblock/ibm-block-csi-operator:1.11.2
```
Comment (kasserater) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:14]: the community points to quay.io for the operator and the non-community to redhat registry?

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,672 @@
...
+    createdAt: "2022-03-31T11:00:00Z"
```
Comment (kasserater) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:15]: maybe we should update this time stamp

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,672 @@
...
+  relatedImages:
+    - name: ibm-block-csi-operator
+      image: quay.io/ibmcsiblock/ibm-block-csi-operator:1.11.2
+  version: 1.11.2
+  replaces: ibm-block-csi-operator.v1.10.0
+  maturity: stable
+  maintainers:
+    - name: IBM Block CSI Team
+      email: csi.block1@il.ibm.com
+  minKubeVersion: 1.23.0
```
Comment (kasserater) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:204]: this should be 1.27.0

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.2/bundle-1.11.2.Dockerfile [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.10-v4.12"
```
Comment (kasserater) [inline deploy/olm-catalog/ibm-block-csi-operator/1.11.2/bundle-1.11.2.Dockerfile:12]: we will support from 4.13 through 4.15

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml [hunk 6]
```diff
@@ -0,0 +1,660 @@
...
+metadata:
+  name: ibm-block-csi-operator.v1.11.2
+  namespace: placeholder
+  annotations:
+    capabilities: "Seamless Upgrades"
+    olm.skipRange: '>=1.6.0 <1.11.2'
+    categories: "Storage,Cloud Provider"
+    certified: "true"
+    containerImage: registry.connect.redhat.com/ibm/ibm-block-csi-operator:1.11.2
+    createdAt: "2022-08-22T11:00:00Z"
```
Comment (kasserater) [inline deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:12]: we should update this timestamp

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml [hunk 7]
```diff
@@ -0,0 +1,660 @@
...
+    - CSI
+  relatedImages:
+    - name: ibm-block-csi-operator
+      image: registry.connect.redhat.com/ibm/ibm-block-csi-operator:1.11.2
+  version: 1.11.2
+  maturity: stable
+  maintainers:
+    - name: IBM Block CSI Team
+      email: csi.block1@il.ibm.com
+  minKubeVersion: 1.23.0
```
Comment (kasserater) [inline deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:194]: should be 1.27.0

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml [hunk 8]
```diff
@@ -0,0 +1,660 @@
...
+                        - matchExpressions:
+                            - key: kubernetes.io/arch
+                              operator: In
+                              values:
+                                - amd64
+                containers:
+                  - name: ibm-block-csi-operator
+                    resources:
+                      requests:
+                        memory: 50Mi
```
Comment (kasserater) [inline deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:629]: let's raise this to 100Mi, like in the community YAML

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.2/metadata/annotations.yaml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.bundle.channels.v1: stable
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: ibm-block-csi-operator
+
+  com.redhat.openshift.versions: v4.10-v4.12
```
Comment (kasserater) [inline deploy/olm-catalog/ibm-block-csi-operator/1.11.2/metadata/annotations.yaml:9]: should be 4.13-4.15

**Review by kasserater — CHANGES_REQUESTED** [review]

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml (hunk 2 above)
Comment (uriziv1) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:14]: This is what happens also at 1.11.0 and 1.12.0 (see: gg "containerImage:").

Comment (kasserater) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:14]: ack, that's what i thought

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/bundle-1.11.2.Dockerfile (hunk 1 above)
Comment (uriziv1) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/bundle-1.11.2.Dockerfile:12]:
```
AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator> gg openshift.versions | grep 1.11.2
deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/bundle-1.11.2.Dockerfile:12:LABEL com.redhat.openshift.versions="v4.13-v4.15"
deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/metadata/annotations.yaml:8:  com.redhat.openshift.versions: "v4.13-v4.15"
deploy/olm-catalog/ibm-block-csi-operator/1.11.2/bundle-1.11.2.Dockerfile:12:LABEL com.redhat.openshift.versions="v4.13-v4.15"
deploy/olm-catalog/ibm-block-csi-operator/1.11.2/metadata/annotations.yaml:9:  com.redhat.openshift.versions: v4.13-v4.15
```

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml (hunk 3 above)
Comment (uriziv1) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:15]: Command to find all "createdAt" under 1.11.2 directories:
```
AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator> for diri in $(find . -name 1.11.2 -type d); do grep -nR createdAt $diri; done
./deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:12:    createdAt: "2024-04-15T11:00:00Z"
./deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:15:    createdAt: "2024-04-15T11:00:00Z"
AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator>
```

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml (hunk 4 above)
Comment (uriziv1) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:204]: Command to find minKubeVersion related to 1.11.2:

```AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator> gg minKubeVersion | grep 1.11.2
deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:204:  minKubeVersion: 1.27.0
deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:194:  minKubeVersion: 1.27.0
AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator>
```

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml (hunk 6 above)
Comment (uriziv1) [reply deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:12]: Done (see comment above).

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml (hunk 7 above)
Comment (uriziv1) [reply deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:194]: Done (see comment above).

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml (hunk 8 above)
Comment (uriziv1) [reply deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:629]: Done.

```
AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator> gg memory: | grep 1.11.2
deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:641:                        memory: 100Mi
deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:644:                        memory: 500Mi
deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:629:                        memory: 100Mi
deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:632:                        memory: 500Mi
```

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.2/metadata/annotations.yaml (hunk 9 above)
Comment (uriziv1) [reply deploy/olm-catalog/ibm-block-csi-operator/1.11.2/metadata/annotations.yaml:9]: Done (see comment above).

**Review by kasserater — APPROVED** [review]

**Pull request merged**